nx-X11/config/cf/Server.tmpl

```diff
@@ -70,3 +70,11 @@ GLX_DEFINES = GlxDefines
 #ifdef XorgVersion
 #include <xorg.tmpl>
 #endif
+
+#ifdef HasXfont2
```

Comment:
Do variables when passed directly when calling make really override any other definitions? Normally that's the case, but I'm not sure how imake.

Yes, that'll probably work (since `FONT_DEFINES` and `XFONTLIB` are always overridden via our main Makefile), but it's difficult to understand that this section here essentially normally does nothing.

Maybe we should provide comments clarifying that this is just a fallback if these variables are not overridden manually.

Meh, but it gets worse: since we override the variables, `HasXfont2` is essentially useless and won't tell what library is actually being used.

This is why I originally suggested to do the check and use `-DHasXfont2=\"YES\"` in the main Makefile.